benchmark/android_world/arbigent_agent.py

```
@@ -0,0 +1,61 @@
+            self._arbigent_bin,
+            "run",
+            "task",
+            goal,
+            "--max-step",
+            str(self._arbigent_max_step),
+            "--max-retry",
+            str(self._arbigent_max_retry),
+        ]
+
+        logging.info("Running Arbigent: %s", " ".join(cmd))
+        result = subprocess.run(cmd, capture_output=True, text=True)
```

Ruff (0.15.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)